ztp/ran-crd/site-config-cr-ex.yaml

```diff
@@ -34,7 +34,7 @@ spec:
       - NTP.server1
       - 10.16.231.22
     diskEncryption:
-      type: "nbde"
+      type: "tpm2+nbde"
```

Comment:
I'm not sure if we wanna change the example
@lack ?

Reply:
fixed